api-server/api-server-common/src/storage/storage_api/mod.rs

```diff
@@ -609,6 +617,16 @@ pub trait ApiServerStorageRead: Sync {
         address: &str,
     ) -> Result<Vec<Id<Transaction>>, ApiServerStorageError>;
 
+    /// Returns a page of transaction IDs that reference this `token_id`, limited to `len` entries
+    /// and with a `tx_global_index` older than the specified value.
+    /// The `tx_global_index` and is not continuous for a specific `token_id`.
```

Comment:
> The `tx_global_index` and is not continuous

It's ironic that this change was made as a response to the review comment "can you plz put more effort into writing documentation comments".